parser/parser.go

```diff
@@ -275,11 +276,13 @@ func (p *Parser) expectPeek(t token.Type) bool {
 	return false
 }
 
-// ErrorContext returns the current line and a pointer to the error position.
+// ErrorLine returns the current line and a pointer to the error position.
 // If prev is true, the error position is relative to the previous token instead of current one.
-func (p *Parser) ErrorContext(prev bool) string {
+func (p *Parser) ErrorLine(prev bool) string {
```

Comment:
This maybe?

```suggestion
func (p *Parser) ErrorLine(previousToken bool) string {
```

Reply:
```diff
diff --git a/parser/parser.go b/parser/parser.go
index b4e47c9..aa386a5 100644
--- a/parser/parser.go
+++ b/parser/parser.go
@@ -277,11 +277,11 @@ func (p *Parser) expectPeek(t token.Type) bool {
 }
 
 // ErrorLine returns the current line and a pointer to the error position.
-// If prev is true, the error position is relative to the previous token instead of current one.
-func (p *Parser) ErrorLine(prev bool) string {
+// If forPreviousToken is true, the error position is relative to the previous token instead of current one.
+func (p *Parser) ErrorLine(forPreviousToken bool) string {
        line, errPos := p.l.CurrentLine()
-       if prev {
-               // When the error about the previous tokem, adjust the position accordingly.
+       if forPreviousToken {
+               // When the error is about the previous token, adjust the position accordingly.
                // (note this doesn't work when the previous token in on a different line -- TODO: improve)
                errPos -= (p.l.Pos() - p.prevPos)
        }
```

Reply:
Better indeed

Reply:
cool it's in #172